sqlalchemy-stubs/ext/asyncio/session.pyi

```diff
@@ -1,54 +1,72 @@
 from typing import Any
 from typing import Callable
+from typing import ContextManager
+from typing import Iterable
+from typing import Iterator
 from typing import Mapping
 from typing import Optional
+from typing import Sequence
 from typing import TypeVar
+from typing import Union
 
-from . import engine as engine
-from .base import StartableContext as StartableContext
-from .engine import AsyncEngine as AsyncEngine
-from ... import util as util
-from ...engine import Result as Result
-from ...orm import Session as Session
-from ...sql import Executable as Executable
-from ...util.concurrency import greenlet_spawn as greenlet_spawn
+from .base import StartableContext
+from .engine import AsyncConnection
+from .engine import AsyncEngine
+from .result import AsyncResult
+from ... import util
+from ...engine import Result
+from ...engine.base import _ExecutionOptions
+from ...orm import Session
+from ...orm.session import _IdentityMap
+from ...sql import ClauseElement
+from ...sql import Executable
 
 _T = TypeVar("_T")
+_TAsyncSession = TypeVar("_TAsyncSession", bound=AsyncSession)
+_TAsyncSessionTransaction = TypeVar(
+    "_TAsyncSessionTransaction", bound=AsyncSessionTransaction
+)
 
 class AsyncSession:
     dispatch: Any = ...
     bind: Any = ...
     binds: Any = ...
```

Comment:
I think these could be improved, but since `bind` was broken for ~6+ month and no-one complained it can go in the nice to have pile :)